src/utils/sdkService.js

```diff
@@ -95,6 +95,15 @@ class SdkService {
 
     return { did, credentials }
   }
+  async getSharableCredentialLink() {
+    const shareLinkParam = {
+      ttl:"10"
```

Comment:
Since we do not have UI part to define it, then let leave it as default, this means that you should skip this parameter, since its suppose to be nullable: true

Reply:
made changes as you said